### PR TITLE
UI for Emailing and Scheduling Reports

### PR DESCRIPTION
**Pull request by procoder05**
Here are the commits for the UI for Emailing and Scheduling Reports, Opened up a PR

### CODE REVIEW

**Comment by procoder05** [comment]
@mbj36 Please review it!

File: app/views/system/schedulereport.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
+<div class="content-container" >
+        <link rel="stylesheet" href="https://formden.com/static/cdn/bootstrap-iso.css" /> 
+            <style>.bootstrap-iso .formden_header h2, .bootstrap-iso .formden_header p, .bootstrap-iso form{font-family: Arial, Helvetica, sans-serif; color: black}.bootstrap-iso form button, .bootstrap-iso form button:hover{color: white !important;} .asteriskField{color: red;}</style>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.6.5/angular.js"></script>
```
Comment (mbj36) [inline app/views/system/schedulereport.html]: Why these files are included here ?

Comment (procoder05) [reply app/views/system/schedulereport.html]: These is a form library cdn!

Comment (mbj36) [reply app/views/system/schedulereport.html]: Why we need another form library ? we are using Angular bootstrap here

Comment (procoder05) [reply app/views/system/schedulereport.html]: I am removing the Jquery and Angular CDN

**Comment by procoder05** [comment]
@mbj36 Please review the PR now!

**Comment by procoder05** [comment]
I have removed the unnecessary CDNs now! Please check if the submission is okay! @mbj36

**Comment by procoder05** [comment]
@mbj36 By formatting the code you mean to beautify it or something else?

**Comment by mbj36** [comment]
@procoder05 yes to beautify it

**Comment by procoder05** [comment]
Okay I will do it right away!

**Pull request closed without merging**